modules/web-console/frontend/app/types/index.ts

```diff
@@ -87,3 +87,11 @@ export type BsSelectDropdownAction = {
 }
 
 export type BsSelectDropdownActions = BsSelectDropdownAction[]
+
+export type Stacktrace = {
+    message: string,
+    stacktrace: [{
+        message: string
+    }],
+    expanded: boolean
```

Comment:
I believe that data and view model should be kept separate. You store `expanded` flag on ` stacktrace-item` instance controller.

Reply:
Expanded flag moved into stacktrace-item controller.